addon/components/image/template.hbs

```diff
@@ -0,0 +1,8 @@
+<img ...attributes
+  class={{this.class}}
+  width={{this.width}}
+  height={{this.height}}
+  src={{this.src}}
+  alt={{this.alt}}
+  title={{this.title}}
+  onError={{this.onError}}>
```

Comment:
new line :)

Reply:
in hbs? :)

Reply:
Disregard, not sure what's with this comment 😄